update.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+#! Exit immediately if any command fails
+set -e
+
+#! Define the repository URL and target directory
+repo_url="https://github.com/Xgorobot/RaspberryPi-CM4.git"
+target_dir="/home/pi/RaspberryPi-CM4-main"
+rc_local_file="/etc/rc.local"
+startup_command="su pi -c 'bash /home/pi/start1.sh'"
+startup_file="/home/pi/start1.sh"
+
+#! Check if the startup command is in /etc/rc.local, and add it if it's not
+if ! grep -qF "$startup_command" "$rc_local_file"; then
+Sudo sed -i "/^exit 0/i $startup_command" "$rc_local_file"
```

Comment:
Should be `sudo`. Code is not aligned properly

Reply:
Fixed that, those got messed up on copy.